### ISSUE TITLE
python37Packages, python38Packages spyder and spyder_3 cannot be build.

### ISSUE THREAD

**Issue by rowanG077** [NixOS/nixpkgs#91484]
**Describe the bug**
the python packages `spyder` and `spyder_3` cannot be build using either `python37Packages` or `python38Packages` due to a missing dependencies.

`python37Packages` with `spyder`:
```
λ nix-shell -p python37Packages.spyder
...
Requirement already satisfied: watchdog in /nix/store/wa3rizns3k02r1jfgx79y92jjz060r1m-python3.7-watchdog-0.10.2/lib/python3.7/site-packages (from spyder==4.1.3) (0.10.2)
ERROR: Could not find a version that satisfies the requirement jedi==0.15.2 (from spyder==4.1.3) (from versions: none)
ERROR: No matching distribution found for jedi==0.15.2 (from spyder==4.1.3)
builder for '/nix/store/6f9mjdbzi54ykj2k13j30i9bjals7k54-python3.7-spyder-4.1.3.drv' failed with exit code 1
error: build of '/nix/store/6f9mjdbzi54ykj2k13j30i9bjals7k54-python3.7-spyder-4.1.3.drv' failed
```

`python37Packages` with `spyder_3`:
```
λ nix-shell -p python37Packages.spyder_3
...
Requirement already satisfied: mccabe<0.7,>=0.6 in /nix/store/qx7249fa29ly5a289x771vhvrq6x5hw6-python3.7-mccabe-0.6.1/lib/python3.7/site-packages (from pylint->spyder==3.3.6) (0.6.1)
ERROR: Could not find a version that satisfies the requirement toml>=0.7.1 (from pylint->spyder==3.3.6) (from versions: none)
ERROR: No matching distribution found for toml>=0.7.1 (from pylint->spyder==3.3.6)
builder for '/nix/store/srgq0w7zh2na7a4qimc5g9v8gzh3h6la-python3.7-spyder-3.3.6.drv' failed with exit code 1
error: build of '/nix/store/srgq0w7zh2na7a4qimc5g9v8gzh3h6la-python3.7-spyder-3.3.6.drv' failed
```

`python38Packages` with `spyder`:
```
λ nix-shell -p python38Packages.spyder
...
Processing ./spyder-4.1.3-py3-none-any.whl
ERROR: Could not find a version that satisfies the requirement python-language-server[all]<0.32.0,>=0.31.9 (from spyder==4.1.3) (from versions: none)
ERROR: No matching distribution found for python-language-server[all]<0.32.0,>=0.31.9 (from spyder==4.1.3)
builder for '/nix/store/p0fvrvym5fvzvzay9addznjxyybir7b8-python3.8-spyder-4.1.3.drv' failed with exit code 1
error: build of '/nix/store/p0fvrvym5fvzvzay9addznjxyybir7b8-python3.8-spyder-4.1.3.drv' failed
```

`python38Packages` with `spyder_3`:
```
λ nix-shell -p python38Packages.spyder
...
Requirement already satisfied: isort<5,>=4.2.5 in /nix/store/f9jfcgj670djsym1g3689kriz4yyn26v-python3.8-isort-4.3.21/lib/python3.8/site-packages (from pylint->spyder==3.3.6) (4.3.21)
ERROR: Could not find a version that satisfies the requirement toml>=0.7.1 (from pylint->spyder==3.3.6) (from versions: none)
ERROR: No matching distribution found for toml>=0.7.1 (from pylint->spyder==3.3.6)
builder for '/nix/store/kdrdp6r7kgr9lc4acrhzl41qly94wfbj-python3.8-spyder-3.3.6.drv' failed with exit code 1
error: build of '/nix/store/kdrdp6r7kgr9lc4acrhzl41qly94wfbj-python3.8-spyder-3.3.6.drv' failed
```

**To Reproduce**
1. Type into the terminal `nix-shell -p python37Packages.spyder` or any of the other variations used above.

**Expected behavior**
I expected the `spyder` dependency to available in the resulting nix-shell.

**Notify maintainers**
@gebner @marcus7070 

**Metadata**

```
λ nix-shell -p nix-info --run "nix-info -m"
these paths will be fetched (0.00 MiB download, 0.00 MiB unpacked):
  /nix/store/0gmrcxw1bmbinqi3hszmmy62qiqrvpmy-nix-info
copying path '/nix/store/0gmrcxw1bmbinqi3hszmmy62qiqrvpmy-nix-info' from 'https://cache.nixos.org'...
 - system: `"x86_64-linux"`
 - host os: `Linux 4.19.37, NixOS, 20.09pre228204.467ce5a9f45 (Nightingale)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3.5`
 - channels(root): `"nixos-20.09pre231796.22a81aa5fc1"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

**Comment by marcus7070** [comment]
Thanks @rowanG077. I think this is the problem here: https://github.com/NixOS/nixpkgs/commit/27f0dabfbfbc15a4233827f08fa36e725f27ca54
I've added `toml` to `propagatedBuildInputs` and it's back to working. I'll make a PR in a minute.

**Comment by gebner** [comment]
This has been fixed in #91489.
